### ISSUE TITLE
'Sites' can't be imported via CSV

### ISSUE THREAD

**Issue by TheRealBecks** [nautobot/nautobot#3541]
### Environment
* Nautobot version (Docker tag too if applicable): Testing environment with `docker-compose` via https://github.com/nautobot/nautobot-docker-compose
```
docker.io/networktocode/nautobot:1.5.9-py3.9 (Nautobot)
docker.io/library/redis:alpine (Redis 6.2.11)
docker.io/library/postgres:13 (PostgreSQL 15.2)
docker.io/networktocode/nautobot:1.5.9-py3.9 (Celery Beat)
docker.io/networktocode/nautobot:1.5.9-py3.9 (Celery Worker)
```

* Python version: 3.9
* Database platform, version: PostgreSQL, Redis
* Middleware(s): None

### Steps to Reproduce
1. Try to import CSV data into `Sites` (http://localhost:8080/dcim/sites/import/):
```
name,slug,status,region,tenant,facility,asn,time_zone,description,physical_address,shipping_address,latitude,longitude,contact_name,contact_phone,contact_email,comments
de.ber.rs,de-ber-rs,Active,,,,,,,,,,,,,,
```

--> not working
2. Create a new site manually with the following data:
```
name: de.ber.rs
slug:  de-ber-rs
status: Active
```

4. Export data
5. Import data --> not working

### What did you expect to happen?
The data can be imported

### What happened instead?
In step 1 and 5 you will get the error message: `Row 1 status: Object not found.`

*Maybe* this is related to #254

**Comment by glennmatthews** [comment]
Thanks!

![image](https://user-images.githubusercontent.com/5603551/230101465-48b040e4-4a48-4579-a139-379db72dc232.png)

For status references in a CSV import, as shown in the screenshot, you should use the `slug` of the status (e.g. `"active"`), not its `name` (`"Active"`), at least as presently implemented. As you pointed out, per #254 export/import are not currently "round trip" capable without modifying the exported CSV before import. We're aiming to fix this in the upcoming 2.0 release.

**Comment by TheRealBecks** [comment]
Thanks, I will try tomorrow! :+1:

**Comment by TheRealBecks** [comment]
@glennmatthews I tried it with the slug for `status` and Nautobot was able to import the data.

Do you need this issue to stay open as being blocked by #254 ?

**Comment by glennmatthews** [comment]
Thanks - I think this falls under the umbrella of #254 and probably doesn't need to stay open as a separate issue.
